config.yaml

```diff
@@ -3,27 +3,12 @@ options:
     type: string
     default:
     description: |
-      The jobbergate-backend-base-url
-  pypi-url:
-    type: string
-    default:
-    description: |
-      PYPI URL
-  pypi-username:
-    type: string
-    default:
-    description: |
-      PYPI USERNAME
-  pypi-password:
-    type: string
-    default:
-    description: |
-      PYPI PASSWORD
+      The jobbergate-backend-base-url  
   version:
     type: string
-    default:
+    default: 1.2.0
```

Comment:
Would 1.* default to highest in version 1?

Reply:
Done